src/redis-cli.c

```diff
@@ -1816,7 +1816,7 @@ static int parseOptions(int argc, char **argv) {
     }
 
     if (!config.no_auth_warning && config.auth != NULL) {
-        fputs("Warning: Using a password with '-a' or '-u' option on the command"
+        fputs("Warning: Using a password with '-a' or '-u' or '--pass' option on the command"
```

Comment:
i'm not sure we wanna mention the `--pass` alias.
p.s. even if we do, i guess the first `or` should be replaced by a `,`

Reply:
Right. After a second thought, not mention the alias seem better. Reverted. 
In this case, i think we can close this one https://github.com/redis/redis/issues/8956